recipes/bvista/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "bvista" %}
+{% set version = "0.1.11" %}
+{% set python_min = "3.8" %}  # Minimum supported Python version on conda-forge
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/b/bvista/bvista-{{ version }}.tar.gz
+  sha256: e3661313133e643fe6a0117fef6e2c2b0892b2c91a9758ab5be322d4bbc9e443
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - bvista = bvista.cli:main
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - flask
+    - flask-socketio
+    - flask-cors
+    - pandas
+    - numpy
+    - requests
+    - matplotlib-base  # prefer this over matplotlib to avoid pulling Qt
+    - seaborn
+    - plotly
+    - missingno
+    - scikit-learn
+    - statsmodels
+    - ipywidgets
+    - pymc
+    - scipy
+    - ipython
+    - dcor
+    - pingouin
+
+test:
```

Comment:
Consider adding a `pip check` test:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L64-L67

Also test the cli:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#test-commands